CHANGELOG.md

```diff
@@ -3,9 +3,12 @@
 All notable changes to this project will be documented in this file. This library adheres to a versioning policy described in [the README](./README.md#versioning). The public API of this library consists of the functions exported in [h3core.js](./lib/h3core.js).
 
 ## [Unreleased]
+
+## [3.0.2] - 2018-07-26
 ### Changed
-- Renamed names of h3.1 or h3-1 to h3 (#4)
 - Updated the core library to v3.0.8 (#10)
+- Renamed names of h3.1 or h3-1 to h3 (#4)
+- Added engine support for Node 10 (#11)
```

Comment:
additions should go under `### Added` (https://keepachangelog.com/en/1.0.0/#how)

Reply:
True, though "added" is actually incorrect here. "Removed Node 10 restriction" or "Changed engines to allow Node 10" would be more accurate. Oh well.